src/components/Banner/index.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
-import './styles.css';
+import './styles.scss';
 
 const Banner = () => {
-    
+    return (
+        <div className='banner'>
+            <img src="https://placehold.co/1440x380?text=Banner" alt="Banner" />
+        </div>
+    )
 };
```

Comment:
For now, we'll be treating the Banner component as an image 'cause it facilitates the layout.